cmake/onnxruntime.cmake

```diff
@@ -60,6 +60,11 @@ function(download_onnxruntime)
       ${PROJECT_BINARY_DIR}/onnxruntime-win-x64-1.14.0.zip
       /tmp/onnxruntime-win-x64-1.14.0.zip
     )
+    # for 32bit x86 program, use below link and HASH value:
+    # set(onnxruntime_URL  "https://github.com/microsoft/onnxruntime/releases/download/v1.14.0/onnxruntime-win-x86-1.14.0.zip")
+    # set(onnxruntime_URL2 "https://huggingface.co/csukuangfj/sherpa-onnx-cmake-deps/resolve/main/onnxruntime-win-x86-1.14.0.zip")
+    # set(onnxruntime_HASH "SHA256=4214b130db602cbf31a6f26f25377ab077af0cf03c4ddd4651283e1fb68f56cf")
+
```

Comment:
Due to lack of CMake knowledge, only provide x86 links and hashs here. Look forward to CMake experts complete dynamically set for x86 and x64 respectively : )

Reply:
Could you use
```cmake
elseif(WIN32)
  if(CMAKE_VS_PLATFORM_NAME STREQUAL Win32)
    # for 32-bit windows
  else()
    # for 64-bit windows
  endif
else()
    message(STATUS "CMAKE_SYSTEM_NAME: ${CMAKE_SYSTEM_NAME}")
    message(STATUS "CMAKE_SYSTEM_PROCESSOR: ${CMAKE_SYSTEM_PROCESSOR}")
    message(FATAL_ERROR "Only support Linux, macOS, and Windows at present. Will support other OSes later")
endif()
```

----

And please refer to
https://k2-fsa.github.io/sherpa/ncnn/install/windows.html
for building on 32-bit windows.

Reply:
@csukuangfj Updated

Reply:
Thanks!

Are you able to use this PR to compile sherpa-onnx on your win32 machine?